_config.yml

```diff
@@ -26,7 +26,7 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: 18F Delivery: Partnership Playbook
+- text: “18F Delivery: Partnership Playbook”
```

Comment:
You'll want to delete the smart quotes (from copy-and-pasting from Slack) by hand and replace them with plain `"` characters.